cli/cmd/lql.go

```diff
@@ -304,11 +327,59 @@ func runQuery(cmd *cobra.Command, args []string) error {
 		err = queryErrorCrumbs(query, err)
 		return errors.Wrap(err, msg)
 	}
-	if data, ok := response["data"]; ok {
-		return cli.OutputJSON(data)
+	// output
+	if err = cli.OutputJSON(response.Data); err != nil {
+		return err
+	}
+	// fail_on_count post
+	if lqlCmdState.FailOnCount != "" {
+		isFail, err := co.isFail(len(response.Data))
+		if err != nil {
+			return err
+		}
+		if isFail {
+			os.Exit(9)
```

Comment:
This is a no-no in regards to design, the CLI should never have random exits, there are other features that will get impacted by this so the recommendation is to create policy enforcement error like https://github.com/lacework/go-sdk/blob/main/cli/cmd/errors.go#L28-L63

Reply:
Fair point regarding random exits.  I saw all this stuff you pointed to, but it seemed like a lot of complexity for what amounts to just wanting a different exit code...  Is there something minimalist here that needs to be satisfied...?

Reply:
@afiune , I got this PR back into shape based on the evolution of the query commands over the last 9 months.  However, we still need to close on the preferred exit workflow per this thread.

Reply:
I forgot about this! Thank you for the reminder. Let me look at this 👀

Reply:
https://github.com/lacework/go-sdk/pull/767